src/SFA.DAS.DigitalCertificates.Application/Queries/GetSharings/GetSharingsQueryHandler.cs

```diff
@@ -0,0 +1,68 @@
+using MediatR;
+using SFA.DAS.DigitalCertificates.Domain.Interfaces;
+using SFA.DAS.DigitalCertificates.Domain.Models;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using static SFA.DAS.DigitalCertificates.Domain.Models.Enums;
+
+namespace SFA.DAS.DigitalCertificates.Application.Queries.GetSharings
+{
+    public class GetSharingsQueryHandler : IRequestHandler<GetSharingsQuery, GetSharingsQueryResult>
+    {
+        private readonly ISharingEntityContext _sharingContext;
+
+        public GetSharingsQueryHandler(ISharingEntityContext sharingContext)
+        {
+            _sharingContext = sharingContext;
+        }
+
+        public async Task<GetSharingsQueryResult> Handle(GetSharingsQuery request, CancellationToken cancellationToken)
+        {
+            var allSharings = await _sharingContext.GetAllSharings(request.UserId, request.CertificateId);
```

Comment:
In GetAllSharings there is OrderBy(CreatedAt) this is used to number the sharings then liveSharings is OrderByDescending(CreatedAt)